rotkehlchen/accounting/structures.py

```diff
@@ -312,6 +312,46 @@ class HistoryEventSubType(SerializableEnumMixin):
     SPEND = 6
     RECEIVE = 7
 
+    def serialize_event_subtype(self) -> str:
+        """Serialize event subtype to a readable string
+        May raise:
+        - DeserializationError
+        """
+        # TODO: change this when the changes to history entries get merged in the left branch
+        if self.name == 'REWARD':
+            return 'get reward'
+        description = EVENTS_SUBTYPES_TO_STR.get(
+            HistoryEventSubType[self.name],
+            UNKNOWN_SUBTYPE,
+        )
+        if description != UNKNOWN_SUBTYPE:
+            return description
+        raise DeserializationError(f'Found staking event with invalid subtype {self.name}')
+
+    @classmethod
+    def deserialize_event_subtype(cls, value: str) -> 'HistoryEventSubType':
+        """Serialize event subtype from redeable string
+        May raise:
+        - DeserializationError
+        """
+        str_to_event_subtype = {v: k for k, v in EVENTS_SUBTYPES_TO_STR.items()}
+        subtype = str_to_event_subtype.get(value, None)
+        # TODO: change this when the changes to history entries get merged in the left branch
+        if subtype is None:
+            return cls.REWARD
+        if subtype != UNKNOWN_SUBTYPE:
```

Comment:
This (subtype == UNKNOWN_SUBTYPE) can never happen here

Reply:
it was a bug, indeed the get above it should return UNKNOWN_SUBTYPE